src/core/shape/vertex.js

```diff
@@ -589,6 +589,7 @@ p5.prototype.endContour = function() {
  *
  * @method endShape
  * @param  {Constant} [mode] use CLOSE to close the shape
+ * @param  {Integer} [count] number of times you want to draw/instance the shape (for WebGL mode).
```

Comment:
Can this be renamed to "instanceCount" or something a little more descriptive?

Reply:
The problem I have with the name `instanceCount` is that it could be taken the wrong way. Perhaps something like `numInstances` would work, but I was thinking of keeping its name simple and understandable since drawing smth a certain amount of times makes sense in my view. If that's a problem then I'd probably change it to `numInstances`, but it might be less understandable rather than more.

Reply:
If u look at the rest of the documentation most of the parameter names are decently simple.

Reply:
I was mainly thinking about the renaming in the context of helping future maintainers understand what this is for. I still think `count` is too ambiguous in this context. `numInstances` sounds fine to me if you prefer that.